Main.cpp

```diff
@@ -82,6 +100,7 @@ int main(int argc, char** argv) {
                            option{"verbose", no_argument, nullptr, 'v'},
                            option{"interval", required_argument, nullptr, 'i'},
                            option{"cgroup-fs", required_argument, nullptr, 'f'},
+                           option{"checkConfig", required_argument, nullptr, 'c'},
```

Comment:
```suggestion
                           option{"check-config", required_argument, nullptr, 'c'},
```

Since we already have `cgroup-config` (note the hypen), it'd be nice if we kept the same naming style.